### PR TITLE
Add eject directive rate limiting

### PR DESCRIPTION
**Pull request by ClifHouck**

### CODE REVIEW

File: arsenal/director/scheduler.py [hunk 1]
```diff
@@ -56,6 +56,16 @@
                default=300,
                help='Determines the amount of time needed to pass before a '
                     'new rate-limit period for cache directives begins.'),
+    cfg.IntOpt('eject_directive_rate_limit',
+               default=0,
```
Comment (pcsforeducation) [inline arsenal/director/scheduler.py]: Is 0 unlimited or disabled?

Comment (ClifHouck) [inline arsenal/director/scheduler.py]: Unlimited - yes, Arsenal will not rate limit ejection directives. Disabled - Yes, rate limiting is disabled if this value is zero.

Comment (pcsforeducation) [inline arsenal/director/scheduler.py]: By disabled I meant disabling ejection.

**Comment by pcsforeducation** [comment]
LGTM +1

**Pull request merged**